bin/gem_dandy

```diff
@@ -10,6 +10,8 @@ require 'date'
 
 Dotenv.load
 
+require 'sentry-raven' if ENV['SENTRY_DSN']
```

Comment:
Feels like a thing `gem_dandy` shouldn't be doing. One of the few areas our (SCF) stack has snuck in,